query_delete.php

```diff
@@ -24,6 +24,8 @@
  * @copyright Copyright 2002  MantisBT Team - mantisbt-dev@lists.sourceforge.net
  * @link http://www.mantisbt.org
  *
+ * @deprecated This page has been replaced by 'manage_filter_delete.php'
```

Comment:
The old pages are still functional.
Note that this PR started as 1.3.x, so decided to not delete immedialty.